doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -520,6 +520,7 @@ Wi-Fi samples
 * :ref:`wifi_radio_test` sample:
 
   * Added capture timeout as a parameter for packet capture.
+  * Increased subcommand scope and corrected subcommand behavior.
```

Comment:
Can it be rephrased as follows?
"Expanded the scope of subcommand and rectified its behavior."

Reply:
Sure, but I'm addressing two different subcommands here. 

_wifi_radio_test show_config_ Increased its scope

_wifi_radio_test tx_pkt_preamble_ rectified its behavior.

Reply:
Ok, then I would suggest to mention the subcommands as well.

Reply:
Something like this?

Expanded the scope of ``wifi_radio_test show_config`` subcommand and rectified the behavior of ``wifi_radio_test tx_pkt_preamble`` subcommand.